frontend/components/FilterDialog.tsx

```diff
@@ -0,0 +1,380 @@
+"use client";
```

Comment:
why is there 2 FilterDialogs?

Reply:
Good catch - this one is a duplicate. Will remove as it has no use in the code base